mailbox/lucene/src/test/java/org/apache/james/mailbox/lucene/search/LuceneMailboxMessageSortOnSentDateTest.java

```diff
@@ -0,0 +1,218 @@
+/****************************************************************
```

Comment:
Because those tests do not pass! I wanted to show you what I was working one, those tests that do not pass was the best way I could find to explain it.
But I didn't have time to introduce the fix in this PR therefore it is a WIP